cli/core/hack/rpm/tanzu-cli.spec

```diff
@@ -0,0 +1,46 @@
+Name:       tanzu-cli
+Version:    %{cli_version}
+Release:    0
+License:    Apache 2.0
+URL:        https://github.com/vmware-tanzu/tanzu-cli/
+Vendor:     VMware
+Summary:    The Tanzu CLI
+
+%ifarch amd64
+%define arch amd64
+%endif
+
+%ifarch aarch64
+# TODO For now, we use the amd64 build for arm64
+%define arch amd64
+%endif
+
+%undefine _disable_source_fetch
+Source0:    https://github.com/vmware-tanzu/tanzu-framework/releases/download/v%{version}/tanzu-cli-linux-%{arch}.tar.gz
+
```

Comment:
```
Provides:       tanzu-cli
Obsoletes:      tanzu-cli  < %{cli_version}
```

would recommend adding these couple of directives for searching / update operations

Reply:
Brilliant!  Will add.  Thanks